db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -172,6 +178,17 @@ abstract class DbCommonsColumnMappers {
     MappedColumnType
       .base[SatoshisPerByte, Long](_.toLong, SatoshisPerByte.fromLong)
   }
+
+  implicit val addressTagMapper: BaseColumnType[AddressTag] = {
+    MappedColumnType
+      .base[AddressTag, String](_.name, tag => UnknownAddressTag(tag))
+  }
+
+  implicit val addressTagTypeMapper: BaseColumnType[AddressTagType] = {
+    MappedColumnType
+      .base[AddressTagType, String](_.name,
+                                    tagType => UnknownAddressTagType(tagType))
+  }
```

Comment:
see comment above, I'd prefer if we had `fromString` methods that only returned `Unknown` if the name is not matched internally